IndicoV2.Abstractions/Submissions/ISubmissionsClient.cs

```diff
@@ -31,7 +31,7 @@ public interface ISubmissionsClient
         /// <param name="resultsFileVersion">Optional. Specifies version to use for the results file.</param>
         /// <param name="cancellationToken"><c><see cref="CancellationToken"/></c> for handling cancellation of asynchronous operations.</param>
         /// <returns><c><see cref="IEnumerable{T}">IEnumerable</see></c> of submissions ids.</returns>
-        Task<IEnumerable<int>> CreateAsync(int workflowId, IEnumerable<(string Name, Stream Content)> files, CancellationToken cancellationToken = default, SubmissionResultsFileVersion? resultsFileVersion = null);
+        Task<IEnumerable<int>> CreateAsync(int workflowId, IEnumerable<(string Name, Stream Content)> files, CancellationToken cancellationToken = default, bool bundle = false, SubmissionResultsFileVersion? resultsFileVersion = null);
```

Comment:
should add `bundle = false` as the last parameter, since it's the newest and callers might be defining `CreateAsync(123, files, token, SubmissionResultsFileVersion.LATEST)` , I think it'll throw a compiler error.

Reply:
ah yeah good call out